### PR TITLE
Northwest Class 4 - Omer Yamak - JS Core 1 Week 2 

### PR DESCRIPTION
**Pull request by omerffaruk**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Omer Yamak
- Your City: Manchester
- Your Slack Name: Omer

# Homework Details

- Module: JS Module 1
- Week: 2

# Notes

- What did you find easy?
It was easy overall, especially exercises in exercise folder were easy
- What did you find hard?
Exercise in extra folder was a little bit harder than other ones
- What do you still not understand?
--
- Any other notes?
--

### CODE REVIEW

**Comment by Anza-Azam** [comment]
Excellent Work Omer, brilliant Job, Easy to understand, and well organized. you've also used good variable and function names.

File: mandatory/3-playing-computer.js [hunk 1]
```diff
@@ -7,36 +7,44 @@
...
+  // Because b in console.log(b) is not defined, it is a parameter (local variable) in f1 and f2 functions, which means it cannot be reached globally, we can only access it inside the f1 and f2 functions
   2. Remove the line that throws the error.
+  // console.log(b) under console.log(a) commented out
   3. What is printed to the console?
   4. How many times is "f1" called?
+  // f1 function will be called three times: explained in for loop inside if 
   5. How many times is "f2" called?
+  // f2 function will be called two times: explained in for loop inside else 
   6. What value does the "a" parameter take in the first "f1" call?
+  // 8
```
Comment (myaskylar) [inline mandatory/3-playing-computer.js:19]: Excellent explanation Omer, We discuss about this exercise together.  Just a little feedback here on question 6 asking what is the value of a in the line
const f1 = function (a, b)   which you comment on line 34

**Comment by myaskylar** [comment]
Great job Omer! Your codes are clean and easy to understand. Well Done! 👍  I've reviewed and leave some comment on your work. Have a look. Thanks.

**Comment by omerffaruk** [comment]
> Excellent Work Omer, brilliant Job, Easy to understand, and well organized. you've also used good variable and function names.

Thank you Anza :) Pleased to hear that

File: mandatory/3-playing-computer.js (hunk 1 above)
Comment (omerffaruk) [reply mandatory/3-playing-computer.js]: > Excellent explanation Omer, We discuss about this exercise together. Just a little feedback here on question 6 asking what is the value of a in the line
> const f1 = function (a, b) which you comment on line 34

Thanks for the feedback Mya :) I reviewed the code and revised it for a cleaner explanation

**Pull request closed without merging**